data/json/construction.json

```diff
@@ -7723,6 +7723,134 @@
+      { "item": "flint", "count": [ 0, 1 ] },
+      { "item": "sharp_rock", "count": [ 0, 1 ] },
+      { "item": "pebble", "charges": [ 0, 4 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_rubble_rock",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_landfill",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [
+      { "item": "glass_shard", "count": [ 0, 4 ] },
+      { "item": "plastic_chunk", "count": [ 0, 1 ] },
+      { "item": "ceramic_shard", "count": [ 0, 1 ] },
+      { "item": "splinter", "count": [ 0, 1 ] },
+      { "item": "scrap_cotton", "count": [ 0, 1 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_rubble_landfill",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_wreckage",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [
+      { "item": "scrap", "charges": [ 0, 3 ] },
+      { "item": "slag", "charges": [ 0, 1 ] },
+      { "item": "scrap_tin", "count": [ 0, 1 ] },
+      { "item": "scrap_aluminum", "count": [ 0, 1 ] },
+      { "item": "scrap_copper", "count": [ 0, 1 ] }
+    ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_wreckage",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_woodchips",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [ { "item": "splinter", "count": [ 0, 3 ] } ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_woodchips",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_ash",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
+    "byproducts": [ { "item": "ash", "charges": [ 0, 50 ] } ],
+    "pre_flags": "RUBBLE",
+    "pre_terrain": "f_ash",
+    "post_terrain": "f_null",
+    "activity_level": "EXTRA_EXERCISE"
+  },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble_radioactive_slag",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "tools": [ [ [ "pickaxe", -1 ], [ "bronze_pickaxe", -1 ], [ "jackhammer", 100 ], [ "elec_jackhammer", 1320 ] ] ],
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": "10 m",
+    "byproducts": [
```

Comment:
You aren't getting discrete uranium and plutonium items out of a mess like this, it would be "radioactive slag items that are hopelessly fused together.

Reply:
agreed, but i wanted to include an item that has LEAK_ALWAYS and RADIOACTIVE to be disposed of
ideally all the items would be radioactive and need disposing or "cleaning"
maybe RADIOACTIVE can work like FILTHY works for clothing, with "craft_inherit" as well

Reply:
![image](https://user-images.githubusercontent.com/115079701/228380047-638359f4-f0b0-445d-a870-25f7749a947c.png)
i can